frontends/README.md

```diff
@@ -31,7 +33,57 @@ Most workspaces are shared dependencies built using typescript and tested with j
 
 ```bash
 # Lint the ol-utilities workspace
-> docker compose run --rm watch yarn workspace ol-utilities run globa:lint-fix
+> docker compose run --rm watch yarn workspace ol-utilities run global:lint-fix
 ```
 
 Again, `global:lint-fix` is defined at the root workspace, not within `ol-utilities`. This works because [yarn commands containing a colon can be run from any workspace](https://yarnpkg.com/getting-started/qa#how-to-share-scripts-between-workspaces).
+
+## Frontend Development
+
+### Docker Compose stack
+
+The frontend and backend stack can be started locally for development with Docker compose:
+
+```bash
+docker compose up
+```
+
+For front end development, at minumum we need these containers.
+
+```bash
+docker compose up nginx web db watch
+```
+
+In this mode, the watch container starts Webpack Dev Server and listens for changes, and building the front end to the local filesystem at `./frontends/mit-open/build`. This is mounted to the `nginx` container for it to serve the static bundle, while routing backend paths to the `web` service.
+
+The application is served at `http://localhost:8063`.
+
+### Local Frontend Dev Server with Local Backend
+
+The `watch` container can be slow to respond to changes to the filesystem mounted onto Docker. If [Hot Module Replacement](https://webpack.js.org/concepts/hot-module-replacement/) is slow for you, we can run Wepback Dev Server directly outside of Docker. Dev Server proxies API requests through to a locally running backend stack.
+
+Run the front end with:
+
+```bash
+yarn watch
```

Comment:
```suggestion
yarn workspace frontends watch
```

Reply:
I've added package scripts at root for these.